clustering/faces/mojarra/pom.xml

```diff
@@ -54,6 +54,8 @@
         <dependency>
             <groupId>jakarta.faces</groupId>
```

Comment:
Just noticed that `org.glassfish` is used as the `groupId` for the `jakarta.faces` `artifactId` in the file above but here `jakarta.faces` is used as the `groupId`.

Is that intentional?